src/admin/export-ireland.ts

```diff
@@ -0,0 +1,60 @@
+import { Brevet } from '../types';
+import { numToDateString, shortYearDateToDate, dateToNum } from '../date';
+import { fetchXlsx } from '../xlsx';
+
+type Raw = {
+  '__rowNum__': string;
+  Date: string;
+  'Event Name': string;
+  Distance: string;
+  Organizer: string;
+  'E-Mail': string;
+  'Organising Club': string;
+  Start: string;
+  Province: string;
+};
+
+const country = 'Ireland';
+
+const XLSX_URL = new URL(
+  'https://www.audaxireland.org/wp-content/uploads/2024/07/Audax-Ireland-Calendar-2024-Excel.xlsx'
```

Comment:
this actually seems like a static file, I wonder if there's something more dynamic, so that it won't need to be changed for 2025? maybe a redirect that goes there or a page that lists it?

Reply:
> this actually seems like a static file, I wonder if there's something more dynamic, so that it won't need to be changed for 2025? maybe a redirect that goes there or a page that lists it?

Hi

They have a calendar in xlsx and pdf formats here: https://www.audaxireland.org/events-calendar/ and also a list of all events posted here: https://www.audaxireland.org/events-calendar/gazetteer/. I decided to use xlsx as the comment said "Please note that the list below contains details of current events, and also many events that are no longer on the schedule". 

It's possible to either match these two lists and periodically check if a new version (2025.. 2026..) appeared on the website. 

Alternatively, I can send an email to the federation asking them if they want to update the way they share calendar -- let say, they can switch to google spreadsheets. Although, this is quite a huge bend.

I'm happy to implement any of these approaches :)

Reply:
Ah, I just got an idea that I can parse the calendar page and if there's a link to xlsx file, parse this file specifically.

This should work :)

Reply:
Yes, that first option looks good. Adding a periodic check probably is overkill right now, but loading that first page and seeing which excel is listed is the way to go I think. Thanks!

Reply:
Please see next two commits. I hope it'll work next year!